tests/ci/common_posix_setup.sh

```diff
@@ -149,6 +149,12 @@ function print_system_and_dependency_information {
     echo "CMake version:"
     cmake --version
   fi
+  if command -v cmake3 &> /dev/null
+  then
+    echo ""
+    echo "CMake version (cmake3 executable):"
+    cmake3 --version
+  fi
```

Comment:
Talked offline, next PR will have logics(`cmake` vs `cmake3`), which is similar to `ninja` vs `ninja-build`.
```sh
  if [[ -x "$(command -v ninja)" ]]; then
    echo "Using Ninja build system (ninja)."
    BUILD_COMMAND="ninja"
    cflags+=(-GNinja)
  elif [[ -x "$(command -v ninja-build)" ]]; then
    echo "Using Ninja build system (ninja-build)."
    BUILD_COMMAND="ninja-build"
    cflags+=(-GNinja)
  else
    echo "Using Make."
    BUILD_COMMAND="make -j${NUM_CPU_THREADS}"
  fi
```